pkg/commands/process/settings/settings.go

```diff
@@ -164,6 +170,7 @@ func (rule *Rule) PolicyType() bool {
 
 func FromOptions(opts flag.Options) (Config, error) {
 	policies := DefaultPolicies()
+	builtInRules := BuiltInRules()
```

Comment:
Do these need to be completely separate? Or could we just load them from a separate location? ie. add them to `loadRules`

Reply:
I thought it was the easiest to keep things separated (at least for now) 
See https://github.com/Bearer/curio/pull/475/files#diff-cf923dbfaf4cadc868828b30bfb30d1fd225d3178248d9521dca28be0504446dR37

Reply:
Fair enough. Maybe you can re-use pieces of `loadRules`? I'd like to avoid the chance of loading things in a different way if possible.